sirepo/template/srw.py

```diff
@@ -848,6 +860,17 @@ def stateful_compute_undulator_length(data):
     return compute_undulator_length(data.args["tabulated_undulator"])
 
 
+def stateless_compute_results_to_activait(data):
+    _SIM_DATA._sim_file_to_other_sim_lib(
```

Comment:
This can't be private, because it is called outside the module. It needs to be public.

Reply:
[987d29e](https://github.com/radiasoft/sirepo/pull/5216/commits/987d29e2fd3306bb306756b8fe3228dad1047030)